core/common/src/annotation/TextStyle.ts

```diff
@@ -0,0 +1,262 @@
+   * Default: "subcategory".
+   */
+  color?: TextStyleColor;
+  /** The name of a font stored in a [Workspace]($backend), used to draw the contents of a [[TextRun]].
+   * Default: "" (an invalid font name).
+   * @note Font names must be unique within a workspace. Uniqueness is semi-case-insensitive per [SQLite's NOCASE collating function](https://www.sqlite.org/datatype3.html#collating_sequences): namely,
+   * the letters A through Z are compared without regard to case, so that "Arial", "arial", and "ARiaL" all refer to the same font.
+   * ###TODO obtain clarity on collation rules.
+   */
+  fontName?: string;
+  /** The height each line of text, in meters. Many other settings use the line height as the basis for computing their own values.
+   * For example, the height and offset from baseline of a subscript [[TextRun]]  are compuated as lineHeight * [[subScriptScale]] and
+   * lineHeight * [[subScriptOffsetFactor]], respectively.
+   * Default: 1.0. */
+  lineHeight?: number;
+  /** Multiplier used to compute the vertical distance between two lines of text.
+   * The distance is computed in meters as lineSpacingFactor * [[lineHeight]].
+   * Default: 0.5.
+   */
+  lineSpacingFactor?: number;
+  /** Specifies whether the content of a [[TextRun]] should be rendered **bold**.
+   * Default: false.
+   */
+  isBold?: boolean;
+  /** Specifies whether the content of a [[TextRun]] should be rendered in *italics*.
+   * Default: false.
+   */
+  isItalic?: boolean;
+  /** Specifies whether the content of a [[TextRun]] should be underlined.
+   * Default: false.
+   */
+  isUnderlined?: boolean;
+  /** Multiplier used to compute the height of both the numerator and denominator of a [[FractionRun]].
+   * The height is computed in meters as stackedFractionScale * [[lineHeight]].
+   * Default: 0.7.
+   */
+  stackedFractionScale?: number;
+  /** Specifies how to separate the numerator and denominator of a [[FractionRun]].
+   * Default: "horizontal".
+   */
+  stackedFractionType?: StackedFractionType;
+  /** Multiplier used to compute the vertical offset from the baseline for a subscript [[TextRun]].
+   * The offset is computed in meters as subScriptOffsetFactor * [[lineHeight]].
+   * Default: -0.15.
+   */
+  subScriptOffsetFactor?: number;
+  /** Multiplier used to compute the height of a subscript [[TextRun]].
+   * The height is computed as subScriptScale * [[lineHeight]].
+   * Default: 2/3
+   */
+  subScriptScale?: number;
+  /** Multiplier used to compute the vertical offset from the baseline for a super [[TextRun]].
+   * The offset is computed in meters as superScriptOffsetFactor * [[lineHeight]].
+   * Default: -0.5.
+   */
+  superScriptOffsetFactor?: number;
+  /** Multiplier used to compute the height of a superscript [[TextRun]].
+   * The height is computed as superScriptScale * [[lineHeight]].
+   * Default: 2/3
+   */
+  superScriptScale?: number;
+  /** Multiplier used to compute the width of each glyph.
```

Comment:
@jffmarker it doesn't seem right to say "width of each glyph", because glyphs can of course vary in width. What should it say instead?

Reply:
Comment is probably over-specific with no benefit... I might go with "Scale factor applied to the width of the glyphs."